schemas/mod-source-record-manager/jobMonitoring.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
Is it really needed to make this schema shared? I assume that it will be used only in SRM.

Reply:
Why not keep it here? It may shared later

Reply:
Not all schemas are shared, only those that are used by other modules. For example `sourceRecordState.json` is not shared because it is needed only in SRM.

Reply:
I know. This schema has a reference to the JobExecution, so it could be potentially used in places where JobExecution is used, the same rule as for JobExecutionProgress, JournalRecord, JobExecutionSourceChunk. So it is always cheaper to keep it here rather than do additional work moving it here later :)

Reply:
The SRM has only once schema for internal usage - Instance.json, nothing else :)